tokio/src/loom/std/atomic_u8.rs

```diff
@@ -2,7 +2,6 @@ use std::cell::UnsafeCell;
 use std::fmt;
 use std::ops::Deref;
 
-/// `AtomicU8` providing an additional `load_unsync` function.
```

Comment:
As `AtomicPtr` - would it be more sensible to just remove this completely?

Reply:
fixed